elixir/tests/test.bats

```diff
@@ -1,11 +1,12 @@
-source "${BATS_TEST_DIRNAME}/../plan.sh"
+TEST_PKG_VERSION="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
 
 @test "Version matches" {
-  result="$(elixir --version | tail -1 | awk '{print $2}')"
-  [ "$result" = "${pkg_version}" ]
+  result="$(hab pkg exec ${TEST_PKG_IDENT} elixir --version | tail -1 | awk '{print $2}')"
+  echo $result
```

Comment:
Is this a debug `echo` that was left in accidentally? If not, maybe we should say

`echo "Version is: ${result}"`